app/src/main/java/com/example/ithaca_transit_android_v2/views/RouteCardCompact.kt

```diff
@@ -0,0 +1,137 @@
+package com.example.ithaca_transit_android_v2.views
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.*
+import androidx.annotation.NonNull
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.ithaca_transit_android_v2.NetworkUtils
+import com.example.ithaca_transit_android_v2.R
+import com.example.ithaca_transit_android_v2.models.Coordinate
+import com.example.ithaca_transit_android_v2.models.Route
+import com.example.ithaca_transit_android_v2.presenters.RouteCardPresenter
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import io.reactivex.disposables.Disposable
+import kotlinx.android.synthetic.main.route_card_compact.*
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.async
+import kotlinx.coroutines.runBlocking
+import kotlin.collections.ArrayList
+
+class RouteCardCompact : AppCompatActivity() {
+
+    val routeCardContext = this
+
+    var rvAdapter = RvAdapter(ArrayList())
+    private lateinit var searchDisposable: Disposable
+
+    var dataList = ArrayList<Route>()
+
+
+    //Temporary data used for networking calls.
+    val end = Coordinate(42.45352923315714, -76.4802676615646)
+    val uid = "E4A0256E-5865-4E9F-8A5A-33747CAC7EBF"
+    val time = 1581308199.0
+    val destinationName = "Bill & Melinda Gates Hall"
+    val start = Coordinate(42.447319055865485, -76.48542509781194)
+    val arriveBy = false
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.route_card_compact)
+
+        runBlocking {
```

Comment:
i'm p sure this `runBlocking` code should be moved to the presenter

Reply:
This code is temporarily in the activity class. I need an observable object to be passed to the presenter for it to then inflate the view and make the networking call. That observable should be made in the activity of another class.

Reply:
leave a comment in the code stating that this is temp